src/main/java/net/minestom/server/item/event/GlobalItemEventManager.java

```diff
@@ -0,0 +1,23 @@
+package net.minestom.server.item.event;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+public class GlobalItemEventManager {
+
+    private static final ConcurrentHashMap<String, ItemEventHandler> events = new ConcurrentHashMap<>();
+
+    public GlobalItemEventManager() {
+
+    }
+
+    public ItemEventHandler get(String id) {
+        if (events.containsKey(id)) return events.get(id);
```

Comment:
This could use one of those compute methods instead of this

Reply:
Outdated :sweat_smile: